apps/src/templates/rubrics/rubricShapes.jsx

```diff
@@ -0,0 +1,18 @@
+import PropTypes from 'prop-types';
+
+export const evidenceLevelShape = PropTypes.shape({
+  understanding: PropTypes.number,
+  teacherDescription: PropTypes.string,
+});
+
+export const learningGoalShape = PropTypes.shape({
+  key: PropTypes.string,
+  learningGoal: PropTypes.string,
+  aiEnabled: PropTypes.bool,
+  tips: PropTypes.string,
+  evidenceLevels: PropTypes.arrayOf(evidenceLevelShape),
+});
+
+export const rubricShape = PropTypes.shape({
+  learningGoals: PropTypes.arrayOf(learningGoalShape),
```

Comment:
Should this also have the lesson / level associated with this rubric?

Reply:
hm, I'm not sure. I'm definitely thinking of this on the teacher side where we might not -- we already know the lesson and level of the page in those cases. Would it be helpful in the curriculum writer case? We can add them into this shape and not use them for the teacher side

Reply:
I think this was a misconception on my part.  I see now that `PropTypes.shape` allows other properties to be included with the object (vs `PropTypes.exact`).  With that, I think we are good here for now!

Reply:
Ooo I didn't know about `PropTypes.exact`! Maybe something for us to try once we get a little farther in development -- I would try it now but I'm having a bit of trouble finding up to date docs on it :/